components/Layout/BaseLayout/BaseLayout.tsx

```diff
@@ -3,7 +3,7 @@ import { ILayout } from "../types"
 
 const BaseLayout = ({ children }: ILayout) => (
   <div
-    className="w-screen h-screen bg-[url('/images/Layout/background.png')] bg-contain
+    className="w-screen h-screen bg-[url('/images/Layout/background-green.png')] bg-contain
```

Comment:
can you please `remove` any images that are no longer being used after your changes? 🚮 
- `background.png`
- any components from original `/` route that are no longer being used